### PR TITLE
Create JPMC CFG Team 4

### PR DESCRIPTION
**Pull request by SinghHarshita**

### CODE REVIEW

**Comment by Rishabh-kanwar** [comment]
Hello

**Comment by Rishabh-kanwar** [comment]
8283824166 this is my number you can call me if you are comfortable or we can chat

File: JPMC CFG Team 4 [hunk 1]
```diff
@@ -0,0 +1 @@
+Hello! Both of us are in team 4 for CFG. Please contact me asap.
```
Comment (mayuri-898) [inline JPMC CFG Team 4:1]: Hello Harshita, Where should I contact you?

**Comment by Rishabh-kanwar** [comment]
https://chat.whatsapp.com/BCm9wmD94pxBlhcqXKH7cz


Use the above link to join the what's app group

File: JPMC CFG Team 4 (hunk 1 above)
Comment (SinghHarshita) [reply JPMC CFG Team 4:1]: https://chat.whatsapp.com/BCm9wmD94pxBlhcqXKH7cz

Use this link to join the group

**Pull request closed without merging**